src/display/editor/tools.js

```diff
@@ -971,6 +971,22 @@ class AnnotationEditorUIManager {
     );
   }
 
+  get highlightColors() {
+    const { _highlightColors } = this;
+    if (!_highlightColors) {
+      return shadow(this, "highlightColors", null);
+    }
+    const map = new Map();
+    const hasHCM = !!this.#pageColors;
+    for (const [name, color] of _highlightColors) {
+      if (hasHCM === name.endsWith("_HCM")) {
```

Comment:
It seems easier to me to write this as `if (hasHCM && name.endsWith("_HCM")) {` to make it more clear that we check two conditions here.